src/interpolate/Interpolate_transform.re

```diff
@@ -0,0 +1,76 @@
+type cssTransform = {
+  transform: string,
+  transformProperty: Js.Nullable.t(string),
+};
+
+[@bs.deriving jsConverter]
+type transformProperties = [
+  | `translate
+  | `translateX
+  | `translateY
+  | `translateZ
+  | `skew
+  | `skewX
+  | `skewY
+  | `skewZ
+  | `rotate
+  | `rotateX
+  | `rotateY
+  | `rotateZ
+  | `scale
+  | `scaleX
+  | `scaleY
+  | `scaleZ
+  | `perspective
+];
```

Comment:
Using our polymorphic variants once again! This helps us to ensure that the user has actually passed us a valid property to use with `transform`. I'm leaving `matrix` out for now – too complicated for a Wednesday 😂